ktor-client/ktor-client-core/js/src/io/ktor/client/engine/js/browser/BrowserFetch.kt

```diff
@@ -31,13 +32,13 @@ internal fun CoroutineScope.channelFromStream(
             channel.flush()
         }
     } catch (cause: Throwable) {
-        reader.cancel(cause)
+        reader.cancel(cause).catch { /* ignore */ }.await()
         throw cause
     }
 }.channel
 
 internal suspend fun ReadableStreamDefaultReader<Uint8Array>.readChunk(): Uint8Array? =
-    suspendCancellableCoroutine { continuation ->
+    suspendCoroutine { continuation ->
```

Comment:
Made this coroutine non-cancellable to ensure the exception thrown on `controller.abort()` is propagated to the upper level.